### DISCUSSION TITLE
Is there a way to set favicon for Uno WebAssembly application?

### DISCUSSION THREAD

**Discussion by HavenDV** [unoplatform/uno#8094] [Q&A] [answered]
Is there a way to set favicon for Uno WebAssembly application?

P.S. Thanks for the answer, it's trivial.
```xml
<ItemGroup>
  <Content Include="favicon.ico" UnoDeploy="Root" />
</ItemGroup>
```
or just add your `favicon.ico` file to `wwwroot` folder.

**Comment by jeromelaban** [comment, marked as answer]
See [this documentation](https://github.com/unoplatform/Uno.Wasm.Bootstrap/blob/main/doc/features-additional-files.md#support-for-additional-content-files) for adding files at the root of the dist folder.

**Reply by agneszitte** [reply]
@jeromelaban We still need to add some documentation regarding the way to set the favicon for WebAssembly for https://platform.uno/docs/articles/intro.html right? (as of right now there is nothing).

**Reply by jeromelaban** [reply]
I don't think this is an intro-level topic, but we could set it by default in the templates, though.

**Reply by agneszitte** [reply]
@jeromelaban sorry I was not clear enough (nothing related to intro-level topic), meaning there is nothing right now regarding favicon in the entire documentation for Uno `platform.uno/docs`. 
set it by default in the templates? meaning having a default uno favicon that can be replaced by the dev after for their custom one?

**Reply by jeromelaban** [reply]
Yes, there's no documentation indeed. We should set it by default in the template meaning we should have one provided using the uno logo in the default template.

**Reply by agneszitte** [reply]
Agree, @jeromelaban! Will keep this aside when we have time to take care of it. (cc @carldebilly)

**Reply by carldebilly** [reply]
@jeromelaban, BTW the link to doc is dead now...  we should update when published to Uno Docs :-)

**Reply by tomchoondal** [reply]
@agneszitte  I need to change the default uno favicon to a different icon, can anyone help me to change the icon.
